lua/astrocommunity/code-runner/toggletasks-nvim/init.lua

```diff
@@ -0,0 +1,15 @@
+return {
+  "jedrzejboczar/toggletasks.nvim",
+  dependencies = {
+    { "nvim-lua/plenary.nvim" },
+    { "akinsho/toggleterm.nvim" },
+    { "nvim-telescope/telescope.nvim" },
+  },
+  opts = {},
+  init = function() require("telescope").load_extension "toggletasks" end,
```

Comment:
```suggestion
  init = function()
    require("astrocore").on_load("telescope.nvim", function() require("telescope").load_extension "toggletasks" end)
  end,
```

respect the lazy loading of telescope